include/grpcpp/ext/otel_plugin.h

```diff
@@ -106,6 +106,9 @@ class OpenTelemetryPluginBuilder {
   /// options can be added.
   OpenTelemetryPluginBuilder& AddPluginOption(
       std::unique_ptr<OpenTelemetryPluginOption> option);
+  // Records \a optional_label_key on all metrics that provide it.
+  OpenTelemetryPluginBuilder& AddOptionalLabel(
```

Comment:
I'm wondering if this should be guarded by an `experimental_type`

```
class experimental_type {
public:
    explicit experimental_type(OpenTelemetryPluginBuilder* builder) : builder_(builder) {}
// Records \a optional_label_key on all metrics that provide it.
  OpenTelemetryPluginBuilder& AddOptionalLabel(absl::string_view optional_label_key);
};

/// NOTE: The function experimental() is not stable public API. It is a view
  /// to the experimental components of this class. It may be changed or removed
  /// at any time.
  experimental_type experimental() { return experimental_type(this); }
```

Reply:
What do you think @markdroth ?

Reply:
I don't think it needs to be experimental.  It seems like this will be a pretty basic use-case that we will want to support.